ttnn/cpp/ttnn/operations/conv/conv2d/device/kernels/reader_conv_activations_2d_mcast_padded_with_halo_3x3_weights_v2.cpp

```diff
@@ -158,7 +159,9 @@ void kernel_main() {
             uint32_t two_reader_indices = packed_reader_indices_ptr[reader_idx];
             #if DILATION_W == 1
             read_channels(l1_write_addr_act, act_l1_read_addr, two_reader_indices & 0xffff, conv_act_c_read_bytes, coalesced_read_bytes, stride_h_bytes);
+            l1_write_addr_act += act_block_w_extra_align_bytes;
```

Comment:
can we avoid doing this add if align is zero? with constexpr

Reply:
Done.